src/index.js

```diff
@@ -120,7 +141,7 @@ function App() {
                 </thead>  
 
                 <tbody>  
-                    <ToDoItems todos={todos} toggleCompleted={toggleCompleted} />
+                    <ToDoItems todos={renderList()} toggleCompleted={toggleCompleted} />
```

Comment:
This is very dangerous. Create a new variable to store the value. Never ever execute the function in props. This could be okay now but this is an habit that if you don't be careful you could easily crash your app.